src/components/LatestChanges/index.js

```diff
@@ -1,171 +1,135 @@
-        <div className="download-container">
-          <div>
-            <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-0">
-              Photo functionality user guide
-            </h2>
-            <p className="govuk-body-m govuk-!-margin-bottom-0 govuk-!-margin-top-5">
-              This is a step-by-step guide to the new functionality, with
-              annotated screenshots of the process
-            </p>
-          </div>
-        </div>
-
-        <a
-          download="User-guide-Photo-functionality-in-Repairs-Hub"
-          className="govuk-button lbh-button govuk-!-margin-top-3=5"
-          href="https://drive.google.com/file/d/1F4UUqqG6ctqvhKXgFIAff2fba_myF0i0/view?usp=sharing"
-        >
-          Download
-        </a>
-      </div>
-
       <div className="govuk-body">
         <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
           What’s changing and when?
         </h2>
         <p className="govuk-!-margin-bottom-5">
-          We are introducing new functionality into Repairs Hub so you can add
-          and view photos to a work order.
+          We’re introducing new functionality so operatives can now report
+          Follow Ons on Repairs Hub.
         </p>
         <p className="govuk-!-margin-bottom-5">
-          This change will be live in Repairs Hub from Tuesday 6 August 2024.
+          This change will be live on{' '}
+          <strong>Thursday December 5th 2024.</strong>
         </p>
         <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
-          How will it impact how I do my job?
+          What is changing?
         </h2>
         <p className="govuk-!-margin-bottom-5">
-          We are asking <strong>operatives</strong> to add relevant photos of
-          the job when closing a work order:
+          When you close a work order, you will be able to record a follow on
+          within the app instead of having to use the google form.
+          <br />
+          To start the process you just need to tap that ‘Further work is
+          required’ which will appear after you click ‘Visit completed’ when
+          closing the work order:
         </p>
-        <ul className="govuk-list govuk-list--bullet">
-          <li>
-            This is for all circumstances - whether the repair is complete, if
-            there is further work to be carried out, or if it is a no access
-            job.
-          </li>
-          <li>
-            Operatives can also add photos after they’ve closed a job using RH
-            desktop.
-          </li>
-          <li>
-            We recommend you take photos of the work as you go (eg. before,
-            during and after) and can then add the photos from your Photo
-            Library.
-          </li>
-        </ul>
+
+        <Image
+          src={closeWorkOrder}
+          width={300}
+          height={300}
+          alt="Close work order"
+        />
```

Comment:
I would recomend setting both width and height. I would set one or the other, so the image is responsive

Reply:
done